jest.config.js

```diff
@@ -14,4 +14,5 @@ module.exports = createConfig('jest', {
   moduleNameMapper: {
     '^lodash-es$': 'lodash',
   },
+  modulePathIgnorePatterns: ["<rootDir>/.*/__mocks__"],
```

Comment:
Why was this needed? (I'm not very familiar with jest configs)

Reply:
I was creating multiple `index.js` files in the various `__mock__` folders, raising a warning that mocks must have unique names. I think removing these `index.js` files will be the proper solution.